src/app/inventory/ItemIcon.m.scss

```diff
@@ -92,6 +92,12 @@ $commonBg: #366f42;
   height: calc(var(--item-size) - #{2 * $item-border-width});
   width: calc(var(--item-size) - #{2 * $item-border-width});
   position: absolute;
+
+  // Ensure that the contained image is correctly scaled down in cases where --item-size is reduced
+  // e.g. Loadout Optimizer Exotic armor picker, Loadout Fashion Mods, Loadout Organizer perk columns
+  & > img {
+    width: 100%;
+  }
```

Comment:
Not sure I understand how this would work - the iconOverlay class is applied to the `img` itself now, it shouldn't have an `img` as a child.

If `--item-size` is changed, why wouldn't the existing calculations based on `--item-size` handle it?

Reply:
Ahh, good catch. The real reason this broke is that `ItemIcon`'s DOM was flattened but `DefItemIcon`'s was not. I've reverted the above scaling change, removed the `div` that surrounds the overlay image (same change as what was done for #8478) and confirmed the issue is fixed.